### PR TITLE
More pygnmi examples and source code

### PR DESCRIPTION
**Pull request by ksator**

### CODE REVIEW

File: src/pygnmi/capabilities.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import json
+
+host = ('10.73.1.105', '6030')
+username ='arista'
+password ='arista'
+
+with gNMIclient(target=host, username=username, password=password, insecure=True) as gc:
+    result = gc.capabilities()
+    pp(result)
+    #print (json.dumps(result, sort_keys=True, indent=2))
```
Comment (fredhsu) [inline src/pygnmi/capabilities.py:12]: Remove this commented line of code?

File: src/pygnmi/update.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    print(result)
+
+print ('\nGET RPC, interface Ethernet1 config, after the update')
+
+paths=['openconfig-interfaces:interfaces/interface[name=Ethernet1]/config']
+
+with gNMIclient(target=host, username='arista', password='arista', insecure=True) as gc:
+    raw_data = gc.get(path=paths, encoding='json_ietf')
+    print(json.dumps(raw_data, sort_keys=True, indent=2))
+    # print(raw_data['notification'][0]['update'][0]['val']['openconfig-interfaces:description'])
```
Comment (fredhsu) [inline src/pygnmi/update.py:44]: remove commented code?

File: src/pygnmi/capabilities.py (hunk 1 above)
Comment (ksator) [reply src/pygnmi/capabilities.py]: done, thanks

File: src/pygnmi/update.py (hunk 2 above)
Comment (ksator) [reply src/pygnmi/update.py]: done, thanks

**Review by fredhsu — APPROVED** [review]
lgtm

**Pull request merged**